ddtrace/llmobs/_trace_processor.py

```diff
@@ -43,8 +43,9 @@ class LLMObsTraceProcessor(TraceProcessor):
     Processor that extracts LLM-type spans in a trace to submit as separate LLMObs span events to LLM Observability.
     """
 
-    def __init__(self, llmobs_span_writer):
+    def __init__(self, llmobs_span_writer, evaluators=None):
         self._span_writer = llmobs_span_writer
+        self._evaluators = evaluators
```

Comment:
If we're going to have evaluators be a part of the trace processor, we're going to need to ensure we do the same thing on `_child_after_fork()` as we do for the span writer, i.e. something like
```
self._evaluator = self._evaluator.recreate()
self._trace_processor._evaluator = self._evaluator
```